pkg/allocator/allocator.go

```diff
@@ -1074,7 +1080,7 @@ func (a *Allocator) NewRemoteCache(remoteName string, remoteAlloc *Allocator) *R
 // kvstore will be maintained in the RemoteCache structure returned and will
 // start being reported in the identities returned by the ForeachCache()
 // function. RemoteName should be unique per logical "remote".
-func (a *Allocator) WatchRemoteKVStore(ctx context.Context, rc *RemoteCache, onSync func(context.Context)) {
+func (a *Allocator) WatchRemoteKVStore(ctx context.Context, rc *remoteCache, onSync func(context.Context)) {
```

Comment:
This could be unexported as well while at it, it's only used in the package and it would be weird to have an exported method take an argument of an unexported type.

Reply:
Yes. Thanks. Done.